maestrowf/interfaces/script/localpooladapter.py

```diff
@@ -0,0 +1,204 @@
+import logging
+import os
+
+from maestrowf.abstracts.interfaces import SchedulerScriptAdapter
+from maestrowf.abstracts.enums import JobStatusCode, State, SubmissionCode, \
+    CancelCode
+from maestrowf.interfaces.script import CancellationRecord, SubmissionRecord
+from pyaestro.utilities.executor import Executor, ExecTaskState, ExecCancel
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LocalPoolAdapter(SchedulerScriptAdapter):
+    """Interface class for the flux scheduler (on Spectrum MPI)."""
```

Comment:
Copy pasted doc string?

Reply:
Actually, scrolling down more, is this whole thing really aimed at using with flux specifically?

Reply:
Oh, that was totally a copy-paste mistake... I used the Flux adapter as a baseline for the API. I'll go back and fix that.